Week2/homework/maartjes_work.js

```diff
@@ -1,47 +1,84 @@
-'use strict';
+"use strict";
 
+// 2.2
 const monday = [
   {
-    name: 'Write a summary HTML/CSS',
+    name: "Write a summary HTML/CSS",
     duration: 180
   },
   {
-    name: 'Some web development',
+    name: "Some web development",
     duration: 120
   },
   {
-    name: 'Fix homework for class10',
+    name: "Fix homework for class10",
     duration: 20
   },
   {
-    name: 'Talk to a lot of people',
+    name: "Talk to a lot of people",
     duration: 200
   }
 ];
 
 const tuesday = [
   {
-    name: 'Keep writing summary',
+    name: "Keep writing summary",
     duration: 240
   },
   {
-    name: 'Some more web development',
+    name: "Some more web development",
     duration: 180
   },
   {
-    name: 'Staring out the window',
+    name: "Staring out the window",
     duration: 10
   },
   {
-    name: 'Talk to a lot of people',
+    name: "Talk to a lot of people",
     duration: 200
   },
   {
-    name: 'Look at application assignments new students',
+    name: "Look at application assignments new students",
     duration: 40
   }
 ];
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+let durationOfTasks = tasks.map(Element => Element.duration);
+
+ console.log(durationOfTasks);
+
+// Map the tasks to durations in hours.
+let durationInHours = durationOfTasks
+  .map(element => element / 60)
```

Comment:
Since you are not chaining array methods here, it is best to place .map()` on the same line as you did in line 48. Since you are mapping durations here, a single elements of that array can be called a `duration`. Therefore that is a more explicit (and therefore) better name for the arrow function argument than the more generic `element`.

ESLint: The `let` can be a `const`.

As the result is the result of the mapping is an array, it is best to reflect that in the name by using a plural for durations: `durationsInHours`.